server/api/v1/admin/game/image/index.delete.ts

```diff
@@ -1,20 +1,31 @@
 import aclManager from "~/server/internal/acls";
 import prisma from "~/server/internal/db/database";
 import objectHandler from "~/server/internal/objects";
+import { type } from "arktype";
 
-export default defineEventHandler(async (h3) => {
+const ModifyGameImage = type({
```

Comment:
I know this wasn't marked for review, but this seems like it should be called DeleteGameImage, not Modify

Reply:
Whoops, forgot to rename that when I copy and pasted